packages/@o3r/pipeline/schematics/ng-add/templates/github/__dot__github/actions/setup/action.yml

```diff
@@ -0,0 +1,21 @@
+name: Setup and Install
+description: Sets up Node.js and installs dependencies
+
+runs:
+  using: "composite"
+  steps:
+    - uses: actions/setup-node@0a44ba7841725637a19e28fa30b79a866c81b0a6 # v4.0.4
+      with:
+        node-version: 20
+        cache: <%= packageManager %>
+    - name: Enable Corepack
+      shell: bash
+      run: corepack enable
+    - name: Install
+<% if (npmRegistry) { %>
```

Comment:
Maybe you should add it commented, or add documentation regarding it, if no `npmRegistry` are provided.
This env are not obvious for people adding registry after generation

Reply:
added some documentation regarding this